bin/generate-fixtures.js

```diff
@@ -37,13 +37,13 @@ branches.forEach((branch) => {
         name: REPO_NAME,
         branch,
         withPullRequestBody: true
-      },
-    }),
+      }
```

Comment:
Seems your fighting prettier new config 😓

Reply:
Sorry i didn't notice all these changes! i'll try and fix this diff tomorrow

Reply:
most likely due to a old yarn install of prettier 🗡️